cli/pkg/release/release.go

```diff
@@ -22,14 +22,26 @@ import (
 
 // a representation of the parameters of the /release endpoint
 type ReleaseParameters struct {
-	Application string
-	Manifests   map[string]string // key is name of the environment and value is the manifest.yaml content
+	Application      string
+	Manifests        map[string]string // key is name of the environment and value is the manifest.yaml content
+	Signatures       map[string][]byte
```

Comment:
(nitpick)
for consistency, we should use the same type.
```suggestion
	Manifests        map[string][]byte // key is name of the environment and value is the manifest.yaml content
	Signatures       map[string][]byte
```